Userland/Libraries/LibWeb/DOMParsing/InnerHTML.cpp

```diff
@@ -27,7 +27,7 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<DOM::DocumentFragment>> parse_fragment(Stri
     }
 
     // 3. Let new children be the result of invoking algorithm given markup, with context set to context.
-    auto new_children = algorithm(context_element, markup);
+    auto new_children = algorithm(context_element, markup, HTML::HTMLParser::AllowDeclarativeShadowRoots::No);
```

Comment:
This change shouldn't be neccessary because I've set a default in HTMLParser.h but the compiler complains if it's missing?

Reply:
I think it's because you're calling it via a function pointer, so it doesn't know what the default values are supposed to be?

Reply:
Ah yeah that's it. Oh well being explicit here isn't the end of the world.